q2lint.py

```diff
@@ -27,6 +28,19 @@ def main():
             if header != HEADER:
                 errors.append('Invalid header: %s' % filepath)
 
+    npm_packages = filter(lambda x: 'node_modules' not in x,
```

Comment:
This filter may not be necessary in production, but it definitely was for local testing. I can drop once functionality seems to be ok.